bin/start_fe.sh

```diff
@@ -63,8 +63,10 @@ check_and_update_max_processes
 #
 # JAVA_OPTS
 # LOG_DIR
+# META_DIR
 # PID_DIR
 export LOG_DIR="$STARROCKS_HOME/log"
+export META_DIR="$STARROCKS_HOME/meta"
```

Comment:
no. be respectful to the configuration set in conf/fe.conf, not overwrite it.

Reply:
the meta dir will be exposed in line 71 `export_env_from_conf`

Reply:
`LOG_DIR`  will also be exposed in line 71 `export_env_from_conf`. Line 68-69 assign the `LOG_DIR` and `META_DIR` to their default values.  Then, in line 71 `export_env_from_conf`, 

https://github.com/StarRocks/starrocks/blob/656a47cbccaa67d5a98907b5327246f62cb2d689/bin/common.sh#L58-L66

if `conf/fe.conf` contains `LOG_DIR = xxx` or `META_DIR = xxx`, then the shell will execute `export LOG_DIR(META_DIR) = xxx`, overwrite the default values in lines 68-69.

Reply:
```
public static String meta_dir = StarRocksFE.STARROCKS_HOME_DIR + "/meta";
```

the var name will be `meta_dir`, not `META_DIR`

Reply:
The comment in `conf/fe.conf` says the uppercase properties are read and exported by bin/start_fe.sh.
https://github.com/StarRocks/starrocks/blob/0a05aff8902686547ea9039cb23d9a44366e8b08/conf/fe.conf#L18-L29

So `LOG_DIR` and `META_DIR` are used by `bin/start_fe.sh`, while `sys_log_dir`, `audit_log_dir`, `meta_dir` are used by `Config.java`.

In `fe/fe-core/src/main/java/com/starrocks/common/Config.java`, I find `sys_log_dir`, `audit_log_dir`, `dump_log_dir`, ... but do not find `LOG_DIR`.

In my opinion, we have 2 solutions:

Solution 1: keep both `meta_dir` and `META_DIR` in `conf/be.conf` for consistency with 
`...log_dir` + `LOG_DIR`.

Solution 2: one `meta_dir` and let `bin/start_fe.sh` parse the lowercase `meta_dir` variable. 

Which solution should we adopt?

Reply:
```
# store metadata, create it if it is not exist.
# Default value is ${STARROCKS_HOME}/meta
# meta_dir = ${STARROCKS_HOME}/meta
```
this is the right one. 

fe.conf can be leveraged to define shell variables for start_fe.sh to use, but itself doesn't take effect unless the real one works which in this case, it is the `meta_dir`.

For example, you can define `META_DIR = /a/b/c` , but the `meta_dir` is actually pointed to `/1/2/3`, so even you `mkdir -p $META_DIR`, that won't help with the start of the fe, where the `/1/2/3` doesn't exist.